### PR TITLE
Internal: Unify build process to include composer install command [ED-15416]

### PR DESCRIPTION
**Pull request by Batsirai-muchareva**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -44,28 +44,39 @@ jobs:
...
+
+          echo "PACKAGE_VERSION=$PACKAGE_VERSION" >> $GITHUB_ENV
+          echo "PLUGIN_FOLDER_FILENAME=$PLUGIN_FOLDER_FILENAME" >> $GITHUB_ENV
+
+      - name: Build plugin
+        uses: ./.github/workflows/build-plugin
+        with:
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+          BUILD_SCRIPT_PATH: "./.github/scripts/build-zip.sh"
+          UPDATE_VERSIONS_SCRIPT_PATH: "./.github/scripts/update-version-elementor-php.sh"
```
Comment (yotamselementor) [inline .github/workflows/build.yml:78]: Why is this required?

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (davseve) [reply .github/workflows/build.yml:78]: The code that updates the plugin version exists in `bump-channel-version` workflow.
https://github.com/elementor/elementor/blob/main/.github/workflows/bump-channel-version/action.yml#L41-L42
Please pay attention that don't increment the version twice in the same build process.

**Review by davseve — COMMENTED** [review]
I see the we took care of the releases workflows only.
I think we should unify the build process among all the workflows, Including Playwright workflows.
So we will have the same behavior in all build processes.

File: .github/workflows/build.yml (hunk 1 above)
Comment (yotamselementor) [reply .github/workflows/build.yml:78]: AFAIU, this PR aligns the build for all CI & release processes, no?
The bumps/versions/whatever are irrelevant for CI, aren't they?

**Review by yotamselementor — APPROVED** [review]

**Review by davseve — APPROVED** [review]
